src/languages/php/frameworks.md

```diff
@@ -0,0 +1,99 @@
+# PHP Featured Frameworks
+
+Platform.sh is the chosen cloud solution behind Symfony, Magento, eZ Systems, and Drupal Commerce. Full Drush support and Composer-based builds make handling dependencies and builds for PHP frameworks as simple as committing your `composer.json` to your project.
+
+## Drupal
+
+Drupal 8 is built around two technologies that make developer's lives easier: Composer and Symfony. Composer simplifies dependency management for PHP applications, and it comes pre-installed on Platform.sh. The default [build flavor](/overview/build-deploy.md#building-the-application) for PHP application runs `composer install` during build, handling all of your dependencies automatically. Platform.sh is also the magic behind Symfony Cloud and Magneto Commerce.
```

Comment:
See above; I don't think we're allowed to mention our white labels.  It's also off topic for the docs as this is not marketing copy; just the fax, sir.